python-checks/src/main/java/org/sonar/python/checks/tests/AssertOnTupleLiteralCheck.java

```diff
@@ -43,17 +44,23 @@ public void initialize(Context context) {
 
         var issue = ctx.addIssue(tuple, MESSAGE);
 
-        if (tuple.leftParenthesis() != null && tuple.rightParenthesis() != null) {
-          // defensive condition
-          issue.addQuickFix(PythonQuickFix.newQuickFix(QUICK_FIX_MESSAGE)
-            .addTextEdit(TextEditUtils.remove(tuple.leftParenthesis()))
-            .addTextEdit(TextEditUtils.remove(tuple.rightParenthesis()))
-            .build());
+        if (isSingletonTupleWithParenthesis(tuple)) {
+          var quickfixBuilder = PythonQuickFix.newQuickFix(QUICK_FIX_MESSAGE)
+            .addTextEdit(TextEditUtils.remove(tuple.leftParenthesis()));
+
+          Token lastComma =  tuple.commas().get(tuple.commas().size() - 1);
+          quickfixBuilder.addTextEdit(TextEditUtils.replaceRange(lastComma, tuple.rightParenthesis(), ""));
+
+          issue.addQuickFix(quickfixBuilder.build());
```

Comment:
Nitpick: maybe we can group the quickfixes statement together
```suggestion
          Token lastComma =  tuple.commas().get(tuple.commas().size() - 1);
          var quickfixBuilder = PythonQuickFix.newQuickFix(QUICK_FIX_MESSAGE)
            .addTextEdit(TextEditUtils.remove(tuple.leftParenthesis()));
            .addTextEdit(TextEditUtils.replaceRange(lastComma, tuple.rightParenthesis(), ""));

          issue.addQuickFix(quickfixBuilder.build());
```